src/components/ValueSet/ValueSetForm.tsx

```diff
@@ -351,7 +351,7 @@ export function ValueSetForm({
     slug: z
       .string()
       .trim()
-      .min(5, t("field_required"))
+      .min(5, t("min_character_validation", { length: 5 }))
       .max(25, t("max_character_validation", { length: 25 }))
```

Comment:
it's generally better to show character range in one error itself:
```suggestion
      .min(5, t("character_count_validation", { min: 5, max: 25 }))
      .max(25, t("character_count_validation", { min: 5, max: 25 }))
```